shell/platform/common/cpp/client_wrapper/include/flutter/binary_messenger.h

```diff
@@ -39,8 +39,14 @@ class BinaryMessenger {
                     const uint8_t* message,
                     const size_t message_size) const = 0;
 
-  // TODO: Add support for a version of Send expecting a reply once
-  // https://github.com/flutter/flutter/issues/18852 is fixed.
+  // Sends a binary message to the Flutter side on the specified channel,
+  // expecting a reply.
+  //
+  // TODO: Consider adding absl as a dependency and using absl::Span.
```

Comment:
You can remove this line (and the copy above); given that this is in a client wrapper now adding a new dependency would be a large burden for anyone using it (this was written back when I thought this class would be part of the library itself).

Reply:
Done